### PR TITLE
Add Signal, Instagram, Farcaster, and Pronouns 

### PR DESCRIPTION
**Pull request by stevenelleman**

### CODE REVIEW

File: apps/frontend/src/pages/profile/edit.tsx [hunk 1]
```diff
@@ -127,6 +133,18 @@ const ProfileEdit = () => {
...
+              placeholder="Signal"
+              variant="secondary"
+              {...register("signalUsername")}
+            />
+            <AppInput
+              label="Instagram"
+              placeholder="Instagram"
+              variant="secondary"
+              {...register("instagramUsername")}
+            />
```
Comment (stevenelleman) [inline apps/frontend/src/pages/profile/edit.tsx:147]: Unfortunately the icon library we use doesn't have signal or instagram icons.

File: apps/frontend/src/pages/profile/overview.tsx [hunk 2]
```diff
@@ -69,6 +70,19 @@ export default function ProfileOverview() {
                 href={`https://twitter.com/${user.userData.twitter.username}`}
               />
             )}
+            {user?.userData.signal?.username && (
+              <CardBox
+                label="Signal"
+                value={user?.userData.signal.username}
+              />
```
Comment (stevenelleman) [inline apps/frontend/src/pages/profile/overview.tsx:77]: Unfortunately Signal doesn't have a username link.

**Comment by stevenelleman** [comment]
image

image

File: apps/frontend/src/lib/storage/types/user/userData/index.ts [hunk 3]
```diff
@@ -27,6 +34,8 @@ export type UserData = z.infer<typeof UserDataSchema>;
 
 export { type TwitterData, TwitterDataSchema } from "./twitterData";
 export { type TelegramData, TelegramDataSchema } from "./telegramData";
+export { type SignalData, SignalDataSchema } from "./signalData";
+export { type InstagramData, InstagramDataSchema } from "./instagramData";
```
Comment (AndrewCLu) [inline apps/frontend/src/lib/storage/types/user/userData/index.ts:38]: export farcaster here?

File: apps/frontend/src/pages/profile/overview.tsx [hunk 4]
```diff
@@ -83,17 +89,37 @@ export default function ProfileOverview() {
...
                 label="Twitter"
-                value={user?.userData.twitter.username}
+                value={`@${user?.userData.twitter.username}`}
                 href={`https://twitter.com/${user.userData.twitter.username}`}
               />
             )}
+            {user?.userData.signal?.username && (
+              <CardBox
+                label="Signal"
+                value={`@${user?.userData.signal.username}`}
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/profile/overview.tsx:106]: Link to signal?

File: apps/frontend/src/pages/profile/overview.tsx [hunk 5]
```diff
@@ -83,17 +89,37 @@ export default function ProfileOverview() {
...
+                label="Instagram"
+                value={`@${user?.userData.instagram.username}`}
+                href={`https://www.instagram.com/${user.userData.instagram.username}`}
+              />
+            )}
+            {user?.userData?.farcaster?.username && (
+              <LinkCardBox
+                label="Farcaster"
+                value={`@${user?.userData.farcaster.username}`}
+                href={`https://www.farcaster.xyz/${user.userData.farcaster.username}`}
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/profile/overview.tsx:120]: Link to warpcast, i.e. https://warpcast.com/andrewlu

File: apps/frontend/src/pages/profile/edit.tsx [hunk 6]
```diff
@@ -108,30 +121,55 @@ const ProfileEdit = () => {
...
+            <AppInput
+              label="Signal"
+              placeholder="Signal"
+              variant="primary"
+              {...register("signalUsername")}
+            />
+            <AppInput
+              label="Farcaster"
+              placeholder="Instagram"
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/profile/edit.tsx:164]: Farcaster*

**Comment by AndrewCLu** [comment]
Tested, adding new socials works + backup after logging out/back in. Two things:

1. link to signal does not show up
2. new social data is not sent to other users upon tap, both direct tap and tap backs

File: apps/frontend/src/pages/profile/overview.tsx (hunk 4 above)
Comment (stevenelleman) [reply apps/frontend/src/pages/profile/overview.tsx:106]: Couldn't find the username link, will look again

Comment (stevenelleman) [reply apps/frontend/src/pages/profile/overview.tsx:106]: sgnl://signal.me/#u/stevenelleman.87 may work

**Comment by stevenelleman** [comment]
image

image

**Comment by stevenelleman** [comment]
Tap backs handled: 

image

File: apps/frontend/src/pages/people/[username].tsx [hunk 7]
```diff
@@ -388,12 +393,15 @@ const UserProfilePage: React.FC = () => {
     );
   }
 
+  console.log("Check pronouns", connection.user.pronouns)
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/people/[username].tsx:396]: oops

File: apps/frontend/src/pages/people/[username].tsx [hunk 8]
```diff
@@ -477,6 +489,46 @@ const UserProfilePage: React.FC = () => {
...
+              {connection?.user?.signal?.username && (
+                <div
+                  onClick={() => {
+                    logClientEvent("user-profile-signal-clicked", {});
+                  }}
+                >
+                  <LinkCardBox
+                    label="Signal"
+                    value={`@${connection.user.signal.username}`}
+                    href={`sgnl://signal.me/#u/${connection.user.signal.username}`}
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/people/[username].tsx:501]: shouldn't this be https lol

File: apps/frontend/src/pages/people/[username].tsx (hunk 7 above)
Comment (stevenelleman) [inline apps/frontend/src/pages/people/[username].tsx:396]: Need to remove, doing now.

**Review by AndrewCLu — APPROVED** [review]

**Pull request merged**